### PR TITLE
[bitnami/postgresql] Add customLivenessProbe and customReadinessProbe

### PR DESCRIPTION
**Pull request by javsalgar**
**Description of the change**

In case we enable extra logging settings, users may want to change the liveness and readiness probes so it doesn't add too much noise in the logs.

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [ ] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/postgresql/templates/statefulset-slaves.yaml [hunk 1]
```diff
@@ -286,6 +286,8 @@ spec:
             timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
             successThreshold: {{ .Values.livenessProbe.successThreshold }}
             failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
+          {{- else if .Values.customLivenessProbe }}
```
Comment (miguelaeh) [inline bitnami/postgresql/templates/statefulset-slaves.yaml:289]: I think this logic is confusing.
To use the customLiveness Probes the user will need to set `livenessProbe.enabled=false` and then `customLivenessProbe`. I would do it like:
```
{{- if and .Values.livenessProbe.enabled .Values.customLivenessProbe }}
...
{{- else if .Values.livenessProbe.enabled  }}
...
{{- else }}
...
{{- end }}
```

And something similar for the readiness probe

**Review by miguelaeh — COMMENTED** [review]
Thank you very much for the PR!!
Please take a look at my comment

File: bitnami/postgresql/README.md [hunk 2]
```diff
@@ -126,6 +126,8 @@ The following tables lists the configurable parameters of the PostgreSQL chart a
...
 | `postgresqlPghbaRemoveFilters`                | Comma-separated list of patterns to remove from the pg_hba.conf file                                                                                                                                                                                                                                                                                                                                                                                                       | `nil`                                                         |
+| `customLivenessProbe`  | Override default liveness probe                                                                                       | `nil`                                                   |
+| `customReadinessProbe` | Override default readiness probe                                                                                      | `nil`                                                   |
```
Comment (miguelaeh) [inline bitnami/postgresql/README.md:130]: Please preserve the format like the other lines of the table.

**Review by miguelaeh — CHANGES_REQUESTED** [review]

File: bitnami/postgresql/templates/statefulset-slaves.yaml (hunk 1 above)
Comment (javsalgar) [reply bitnami/postgresql/templates/statefulset-slaves.yaml:289]: It's already set like that in most of the charts. We have similar logics in other parameters like `externalDatabase`. For consistency I would leave it like the rest.

Comment (javsalgar) [reply bitnami/postgresql/templates/statefulset-slaves.yaml:289]: But I would add it as something to review in the future

Comment (miguelaeh) [reply bitnami/postgresql/templates/statefulset-slaves.yaml:289]: 👍

**Review by miguelaeh — APPROVED** [review]
LGTM!

**Pull request merged**